protoc-gen-openapiv2/internal/genopenapi/template_fuzz_test.go

```diff
@@ -0,0 +1,26 @@
+//go:build go1.18
+// +build go1.18
+
+package genopenapi
+
+import (
+	"regexp"
+	"testing"
+)
+
+var replaceInternalCommentsRegex = regexp.MustCompile(`(?s)(\r?\n)?[ \t]*(\(--)((.*?--\))|.*$)?`)
+
+func FuzzPopulateQueryParameters(f *testing.F) {
```

Comment:
I think this is probably incorrectly named 😬? How about
```suggestion
func FuzzRemoveInternalComments(f *testing.F) {
```

Reply:
Thanks, changed it